assets/scripts/pages/admin/post.js

```diff
@@ -0,0 +1,71 @@
+import $ from "jquery";
+import 'datatables.net-bs5';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/magali-thuaire/blog-magali/pullRequest?prid=8924174)
```suggestion
import "datatables.net-bs5";
```